Problem6/src/problem6/Problem6.java

```diff
@@ -10,7 +13,30 @@
      * @param args the command line arguments
      */
     public static void main(String[] args) {
-        // TODO code application logic here
+        Scanner keyboard = new Scanner(System.in);
+        System.out.print("Type a sentence: ");
+        String input = keyboard.nextLine();
+        input = input.trim();
+
+        StringTokenizer words = new StringTokenizer(input);
+        while (words.hasMoreTokens()) {
+
+            if (words.nextToken().length() > 16) {
+                System.out.println("The input includes argumentativeness, which is longer than 16 characters.");
```

Comment:
If a word is too long, you always output "argumentativeness" instead of the word that was too long